src/services/expenses.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const Expenses = require('../db/models/expenses');
+
+function normalizeExpense({
+  userId,
+  title,
+  category,
+  amount,
+  spentAt,
+  note,
+}) {
+  return {
+    userId,
+    title,
+    category,
+    amount,
+    spentAt,
+    note,
+  };
+}
+
+async function getFilteredExpenses(params) {
+  const { userId, categories, from, to } = params;
+
+  const expenses = await Expenses.findAll();
+
+  return expenses.filter(expense => {
+    const isUserIdMatch = userId
+      ? expense.userId === +userId
```

Comment:
Use 'Number()' instead of the unary plus operator '+' for converting strings to numbers for better readability.

Reply:
https://mate-academy.github.io/style-guides/javascript.html#223-numbers-usenumberfor-type-casting-andparseintalways-with-a-radix-for-parsing-strings